osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -33,6 +33,14 @@ public class ChannelManager : PollingComponent, IChannelPostTarget
         private readonly BindableList<Channel> availableChannels = new BindableList<Channel>();
         private readonly BindableList<Channel> joinedChannels = new BindableList<Channel>();
 
+        // Keeps a list of closed channels. More recently closed channels appear at higher indeces
+        private readonly BindableList<Channel> closedChannels = new BindableList<Channel>();
+
+        // For efficiency purposes, this constant bounds the number of closed channels we store.
+        // This number is somewhat arbitrary; future developers are free to modify it.
+        // Must be a positive number.
+        private const int closed_channels_max_size = 50;
```

Comment:
I think naming convention in this repo has constants using CAPITAL_LETTERS naming convention

Reply:
Bump on this change - other than that, everything else looks good to go :)

Reply:
Jk actually... they're kinda inconsistent on whether these are capitalized or not